src/components/segmentedControl/index.tsx

```diff
@@ -98,6 +102,7 @@ const SegmentedControl = (props: SegmentedControlProps) => {
     outlineColor = activeColor,
     outlineWidth = BORDER_WIDTH,
     throttleTime = 0,
+    segmentsStyle: segmentsStyleProp,
```

Comment:
Why do you need the rename here?

Reply:
There's another `segmentsStyle` in this file

Reply:
Ok...